src/main/java/teamroots/embers/register/BlockRegister.java

```diff
@@ -2,184 +2,373 @@
+    public static final Block STAMPER = new BlockStamper(Material.ROCK, "stamper", true).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block STAMP_BASE = new BlockStampBase(Material.ROCK, "stamper_base", true).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block STIRLING = new BlockStirling(Material.ROCK, "stirling", true).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block STONE_EDGE = new BlockStoneEdge(UNPUSHABLE, "stone_edge", true).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block STONE_VALVE = new BlockStoneValve(UNPUSHABLE, "stone_valve", true).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block STRUCTURE_MARKER = new BlockStructureMarker().setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block TURRET = new BlockTurret(Material.IRON, "turret", true).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block VACUUM = new BlockVacuum(Material.ROCK, "vacuum", true).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block WALL_ASHEN_BRICK = new BlockWallBase(ASHEN_BRICK, "wall_ashen_brick", true).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block WALL_ASHEN_STONE = new BlockWallBase(ASHEN_STONE, "wall_ashen_stone", true).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block WALL_ASHEN_TILE = (new BlockWallBase(ASHEN_TILE, "wall_ashen_tile", true)).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block WALL_CAMINITE_BRICK = new BlockWallBase(BLOCK_CAMINITE_BRICK, "wall_caminite_brick", true).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block WRAPPED_SEALED_PLANKS = new BlockBase(Material.WOOD, "wrapped_sealed_planks", true).setIsFullCube(true).setIsOpaqueCube(true).setHarvestProperties("axe", -1).setHardness(3.1f).setLightOpacity(16);
 
     ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
     private final static List<Block> ALWAYS_REGISTER = Collections.singletonList(
```

Comment:
@Klebestreifen IntelliJ seems to have a problem with the singleton list as it want's to remove all list entries and I have no idea why, maybe you can help me figuring that out.